server/src/main/java/com/unosquare/carmigo/service/JourneyService.java

```diff
@@ -97,4 +110,49 @@ private Journey findJourneyById(final int id) {
     return journeyRepository.findById(id)
         .orElseThrow(() -> new ResourceNotFoundException(String.format("Journey id %d not found.", id)));
   }
+
+  private String prepareRequestToDistanceApi(final CreateCalculateDistanceCriteria criteria) {
+    final StringBuilder request = new StringBuilder("[{\"t\":");
+    request.append("\"").append(criteria.getLocationFrom()).append(",").append(criteria.getCountryFrom()).append("\"");
+    request.append("},{\"t\":");
+    request.append("\"").append(criteria.getLocationTo()).append(",").append(criteria.getCountryTo()).append("\"");
+    request.append("}]");
+    return request.toString();
+  }
+
+  private GrabDistanceDTO convertDistanceHolderToDistanceViewModel(final DistanceHolder distanceHolder) {
```

Comment:
Hi Kleyton!

Do you think you could possibly reduce the size of this method?

I always try and follow the rule of 7.  7 lines per method, 7 methods per class, 7 classes per package.  But that's just me :)